mllib/src/main/scala/org/apache/spark/ml/feature/Binarizer.scala

```diff
@@ -31,10 +31,12 @@ import org.apache.spark.sql.types.{DoubleType, StructType}
  * :: Experimental ::
  * Binarize a column of continuous features given a threshold.
  */
+@Since("1.4.0")
 @Experimental
 final class Binarizer(override val uid: String)
```

Comment:
Also annotate the constructor and the public variables in the constructor, e.g., https://github.com/apache/spark/blob/master/mllib/src/main/scala/org/apache/spark/mllib/feature/IDF.scala#L42. Same for other constructors.